tests/test_service.py

```diff
@@ -168,10 +168,21 @@ def test_rsi_metadata(self):
                 survey["metadata"]["ref_period_end_date"] = "2016-10-31"
                 self.assertValid(survey)
 
+    def test_invalid_version(self):
+        survey = json.loads(self.message['0.0.1'])
+        with self.assertRaises(KeyError):
+            server.ValidSurveyId(survey['survey_id'], '0.0.3')
+
     def test_missing_version(self):
         survey = json.loads(self.message['0.0.1'])
         with self.assertRaises(AttributeError):
-            server.ValidSurveyId('123')
+            server.ValidSurveyId(survey['survey_id'])
+
+    def test_valid_version(self):
+        survey = json.loads(self.message['0.0.1'])
+        value = server.ValidSurveyId(survey['survey_id'], survey['version'])
+        # ValidSurveyId doesn't have a return value, so it returns None
+        self.assertIsNone(value)
```

Comment:
I don't think we need this assert, as all its testing is that a function with no return returns `None` which is standard python behaviour.
The test will be fine in that if it doesn't work there will be an uncaught exception.

Reply:
Ah, fair enough. I wasn't sure if it was okay for there to be a test that didn't have an assert.